pages/kommun/[municipality]/[step].tsx

```diff
@@ -1,23 +1,29 @@
 import { GetServerSideProps } from 'next'
-import dynamic from 'next/dynamic'
 import { useRouter } from 'next/router'
 import { ParsedUrlQuery } from 'querystring'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { ReactElement } from 'react'
+import styled from 'styled-components'
 
 import { ClimateDataService } from '../../../utils/climateDataService'
 import { WikiDataService } from '../../../utils/wikiDataService'
 import { Municipality as TMunicipality } from '../../../utils/types'
 import { PoliticalRuleService } from '../../../utils/politicalRuleService'
 import { ONE_WEEK_MS } from '../../../utils/shared'
-
-const Municipality = dynamic(() => import('../../../components/Municipality/Municipality'))
```

Comment:
Importing dynamically makes no sense since this is a critical code path for this page. I believe removing `dynamic()` here actually will speed up the perceived performance of the municipality pages since they don't have to load the code on the client, but can get it through SSR instead.